src/models/Comment.js

```diff
@@ -3,7 +3,30 @@ export default (sequelize, DataTypes) => {
     comment: {
       type: DataTypes.STRING,
       allowNull: false
+    },
+    userId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'Users',
+        key: 'id'
+      },
+      onUpdate: 'CASCADE',
+      onDelete: 'SET NULL'
+    },
+    requestId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'Requests',
+        key: 'id',
+      },
+      onUpdate: 'CASCADE',
+      onDelete: 'SET NULL'
     }
```

Comment:
well done @torsami77 I think you don't need to repeat the requestId and userId here again since  you already referenced them in the `database/migrations/create-7-comments.js file